Feature/header layout

shashaehhooman opened the pull request:
**Task 1**
Style Menu :
      - Top Menu
      - Sub Menu
      - Main Menu


![Untitled](https://user-images.githubusercontent.com/74895961/187022543-5ad9568b-47bc-48db-b9cc-d65ea1b945f9.png)

shashaehhooman committed:
connect header scss to layout

shashaehhooman committed:
create header.jsx in layout

shashaehhooman committed:
changing every thing

shashaehhooman committed:
Add a file that name headercss

shashaehhooman committed:
div update

shashaehhooman committed:
ok all

shashaehhooman committed:
header and navbar ok

shashaehhooman committed:
header and index of overview changed

shashaehhooman committed:
picture and shoppiong

shashaehhooman committed:
no message

shashaehhooman committed:
add alot of thing that none of your business

rezapct committed:
Update css infrastructure

rezapct committed:
Change topHeader menu style

rezapct committed:
Add font-awsome

shashaehhooman committed:
complete menu

shashaehhooman committed:
test push

shashaehhooman committed:
All header and menu is complete sir

shashaehhooman committed:
changing in header.scss

rezapct committed:
Menu popup style

rezapct committed:
Show bullet beside menu item

rezapct committed:
Search menu

shashaehhooman committed:
Sir I couldnt set false

shashaehhooman committed:
Merge branch 'feature/header_layout' of https://github.com/reza-madankar/shop into feature/header_layout

shashaehhooman committed:
make mistakes

reza-madankar commented:
Thanks for working hard, Please take a look at .... Menu

rezapct committed:
Update Overly action

rezapct committed:
Change search box style

shashaehhooman committed:
***Tasks***
1.All image copy in image file and import from H.D.D
2.All Icon bar are complete.
3.All header.scss format and priority are done.
4.All page review check.

reza-madankar commented:
Please take a look at below items:

- **Assistance menu:**
     -  **All Menu**: Add animation to open and close menu
     -  **Basket Menu**: Add close button + Add mouse hover change color 
     - **Basket Menu**: Add mouse hover to change color 
     - **Basket Menu**: Add mouse hover to change bg-color 
     - **User Menu**: Add mouse hover animation to change space to the left
     - **Search Menu**: remove space between button and textbox
- **Main Menu:** Remove mouse hover animation on title of columns
     

please, Let me know when all of items are fixed

shashaehhooman committed:
***Taks***
1-fix the button of search box(space between button and search filed).
2-fix the hover of note box(color and animation).
3-fix the basket box (hover : color and close button)
***

shashaehhooman committed:
***Tasks***
1-Fix the scss code priority and deatails.
2-Fix the fade in  transition for  bootstrap Navbar.
3-Fix the fade in transition for icons.
***

reza-madankar reviewed (approved):
good job bro

reza-madankar committed:
Merge branch 'main' into feature/header_layout
